tests/options.flags.test.js

```diff
@@ -0,0 +1,76 @@
+const commander = require('../');
+
+// Test the various ways flags can be specified in the first parameter to `.option`
+
+test('when only short flag defined and specified then value is true', () => {
+  const program = new commander.Command();
+  program
+    .option('-p', 'add pepper');
+  program.parse(['node', 'test', '-p']);
+  expect(program.P).toBe(true);
```

Comment:
I'm sorry if I sound picky.
It seems good to use `toBeTruthy`.
https://jestjs.io/docs/ja/expect#tobetruthy

Reply:
No problem. I did consider `toBeTruthy`, but I decided important to test for `true` in at least some places. Say if we changed the implementation to return `1` or `default` or some other truthy value, some of the tests should fail because it would break some clients.

```